.github/workflows/intel-freshness-production-closeout.yml

```
@@ -0,0 +1,125 @@
+        shell: bash
+        run: |
+          set -euo pipefail
+          for key in NEXT_PUBLIC_SUPABASE_URL NEXT_PUBLIC_SUPABASE_ANON_KEY SUPABASE_SERVICE_ROLE_KEY CRON_SECRET; do
+            if [ -z "${!key:-}" ]; then
+              echo "::error::$key is not configured; refusing briefing regeneration."
+              exit 1
+            fi
+          done
+      - name: Start exact built app for bounded synthesis
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:4:1: attempt appears unused. Verify use (or export if used externally)

(shellcheck)